core/domain/prod_validation_jobs_one_off_test.py

```diff
@@ -13062,6 +13062,68 @@ def test_missing_edited_exploration_model_failure(self):
         run_job_and_check_output(self, expected_output, sort=True)
 
 
+class UserAuthModelValidatorTests(test_utils.GenericTestBase):
+
+    def setUp(self):
+        super(UserAuthModelValidatorTests, self).setUp()
+
+        self.signup(USER_EMAIL, USER_NAME)
+        self.user_id = self.get_user_id_from_email(USER_EMAIL)
+        self.gae_id = self.get_gae_id_from_email(USER_EMAIL)
+        user_models.UserAuthModel(id=self.user_id, gae_id=[self.gae_id]).put()
+        self.model_instance = user_models.UserAuthModel.get_by_id(
+            self.user_id)
+        self.job_class = (
+            prod_validation_jobs_one_off
+            .UserAuthModelAuditOneOffJob)
```

Comment:
Move to the line above--this is below the character cut-off so it doesn't need to be wrapped.

Reply:
Done.